lib/AppInfo/Application.php

```diff
@@ -80,7 +80,8 @@ private function connectSearch() {
 	 * Register admin settings
 	 */
 	public function registerAdminPage() {
-		\OCP\App::registerAdmin($this->getContainer()->getAppName(), 'templates/settings');
+		if (\OCP\Util::getVersion() < 14)
```

Comment:
I don't think you need this:
1) `.\occ app:check` will fail when it sees `\OCP\App::registerAdmin` (as it is deprecated), and
2) Even v11 (which is our minimum version) supports <settings><admin>, as stated [here](https://docs.nextcloud.com/server/11/developer_manual/app/info.html)

Can we just completely remove `registerAdminPage` method?

Reply:
> Can we just completely remove registerAdminPage method?

Of course.. Just leave it, because in all your commits you are working in parallel, leaving the previous code. 
For my part, I want to leave behind all vestige of the python tool ..